doc/RecordingControl.xml

```diff
@@ -1595,6 +1595,85 @@ Change Request 2061, 2063, 2065, 2109</revremark>
         </varlistentry>
       </variablelist>
     </section>
+    <section>
+      <title>ExportRecordedSegments</title>
+      <para>Exports the selected recorded segments (from existing recorded data) to the storage attached to the given recording job.</para>
+      <para>The StoragePath shall be the path as defined in the Object Storage annex appended to the StorageUri from the StorageConfiguration.</para>
+      <variablelist role="op">
+        <varlistentry>
+          <term>request</term>
+          <listitem>
+            <para role="param">Time [tt:DateTimeRange]</para>
+            <para role="param">JobToken [tt:RecordingJobReference]</para>
```

Comment:
During the Krakow F2F, members have raised the question why we are not using RecordingConfiguration tokens instead. The issue we see is that without the JobToken, there is no way for the client to specify which Source (media profile) we want to list or export recorded segments for.

Reply:
Based on feedback received, updated the RecordingConfiguration to include the VideoSourceToken inside the RecordingConfiguration and use the RecordingToken instead.